packages/features/bookings/lib/handleNewBooking/test/fresh-booking.test.ts

```diff
@@ -224,6 +233,7 @@ describe("handleNewBooking", () => {
           organizer,
           emails,
           iCalUID,
+          destinationEmail: organizerDestinationCalendarEmailOnEventType,
```

Comment:
Added test here for checking if email is sent to "destination calendar integration" email.